app/src/main/res/drawable/background_gradient.xml

```diff
@@ -5,6 +5,6 @@
         android:startColor="@color/colorPrimary"
         android:centerColor="@color/colorPrimary"
         android:centerY="0.2344"
-        android:endColor="@color/colorPrimaryDark"
+        android:endColor="@color/colorAccent"
```

Comment:
if this is part of the design, we should talk about it with designers. If gradien end color is the same as floating button color, it is not visible very well